src/main/java/com/jungdam/invitation/application/InvitationService.java

```diff
@@ -30,4 +31,9 @@ public boolean existsByAlbumAndTargetMemberAndStatus(Album album, Member member,
         Status status) {
         return invitationRepository.existsByAlbumAndTargetMemberAndStatus(album, member, status);
     }
+
+    @Transactional(readOnly = true)
+    public List<Invitation> findAllByTargetMemberAndStatus(Member member) {
```

Comment:
Status가 고정된 값이면 
```suggestion
    public List<Invitation> findAllByTargetMemberAndPending(Member member) {
```
이런식으로 명시해주는게 어떨까용?

Reply:
findAllByTargetMemberAndPendingStatus로 변경했습니다!